build/yamls/flow-visibility/base/clickhouse.yml

```diff
@@ -36,7 +36,100 @@ spec:
               volumeMounts:
                 - name: clickhouse-configmap-volume
                   mountPath: /docker-entrypoint-initdb.d
+                - name: clickhouse-storage-volume
+                  mountPath: /var/lib/clickhouse
           volumes:
             - name: clickhouse-configmap-volume
               configMap:
                 name: $(CLICKHOUSE_CONFIG_MAP_NAME)
+            - name: clickhouse-storage-volume
+              emptyDir:
+                medium: Memory
+                sizeLimit: 8Gi
```

Comment:
should we use a smaller default value for folks testing this locally?

Reply:
I have some tests on the Clickhouse throughput, which is the maximum number of inserted records per second it can deal without an error. The ClickHouse server can reach the throughput at around 4.3K records per second with a 8Gi deployment, 2K with 4Gi and 800 with 2Gi. 
We select 8Gi from the consideration that it provides a reasonable throughput. Do you have other suggestions of the choice? cc @salv-orlando on this discussion.

Reply:
Updated in document

Reply:
awesome!